node/src/chain_spec.rs

```diff
@@ -148,6 +148,7 @@ pub fn local_config() -> Result<ChainSpec, String> {
     )
     .with_name("NH Local")
     .with_id("nh_local")
+    .with_protocol_id("nh_local")
```

Comment:
We can use `zen` or `tzen` or `nzen` .... seams more close to what polkadot do

Reply:
Done